pkg/controller/v1alpha2/core/workloads/containerizedworkload/containerizedworkload_controller_helper.go

```diff
@@ -75,6 +75,22 @@ func (r *Reconciler) renderService(ctx context.Context,
 	return service, nil
 }
 
+// create ConfigMaps for ContainerConfigFiles
+func (r *Reconciler) renderConfigMaps(ctx context.Context,
+	workload *v1alpha2.ContainerizedWorkload) ([]*corev1.ConfigMap, error) {
+	configMaps, err := TranslateConfigMaps(ctx, workload)
+	if err != nil {
+		return nil, err
+	}
+	for _, cm := range configMaps {
+		// always set the controller reference so that we can watch this configmap and it will be deleted automatically
+		if err := ctrl.SetControllerReference(workload, cm, r.Scheme); err != nil {
```

Comment:
what will happen for this line?

Reply:
It will append the workload as an ownerRef  to the configMap, maybe I should use the deployment instead of workload.